nixos/modules/services/networking/frr.nix

```diff
@@ -106,6 +106,14 @@ let
           TCP Port to bind to for the VTY interface.
         '';
       };
+
+      extraOptions = mkOption {
+          type = types.str;
```

Comment:
This should be an attr of strings to easily supply multiple options.

Reply:
hmmm, not sure

Reply:
Yeah... maybe `with types; listOf str;`.

Reply:
Updated to list of strings